server/routes/polls/polls.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
+const router = express.Router();
+const authentication = require('../../middleware/authentication');
+const PollsMongoModel = require('../../models/Polls');
+const { cloudinary } = require('../../cloudinary/cloudinary');
+const { uuid } = require('uuidv4');
+
+router.post('/create', authentication, async (req, res) => {
+  try {
+    const userId = req.user.user.id;
```

Comment:
I think the userId is accessible by using req.user.id, not req.user.user.id. @TheAllen

Reply:
yeah since I was not able to check my endpoint so not sure on that..... but today will talk to allen let see